### PR TITLE
[cookbook]: Url summarizer

### PR DESCRIPTION
**Pull request by Anindyadeep**
This PR adds the second example: Creating a simple URL content summarizer using PremAI and LangChain.

### CODE REVIEW

File: cookbook/url-topic-summarizer.mdx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+```txt
+streamlit 
+premai
+langchain
+langchain-community
+BeautifulSoup4
+tiktoken
+transformers
+```
```
Comment (nsosio) [inline cookbook/url-topic-summarizer.mdx:35]: no need to list all requirements (they may change and this info is not updated), at most a link to the requirements.txt

File: cookbook/url-topic-summarizer.mdx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+from langchain.chains.llm import LLMChain
+from langchain_core.prompts import PromptTemplate
+from langchain_text_splitters import CharacterTextSplitter
+from langchain.chains.combine_documents.stuff import StuffDocumentsChain
+from langchain.chains import MapReduceDocumentsChain, ReduceDocumentsChain
+
+# Set all the settings here 
+premai_api_key = st.secrets.premai_api_key
+premai_project_id = 4071
```
Comment (nsosio) [inline cookbook/url-topic-summarizer.mdx:80]: avoid real project_ids

File: cookbook/url-topic-summarizer.mdx (hunk 1 above)
Comment (Anindyadeep) [reply cookbook/url-topic-summarizer.mdx:35]: Yeah that sounds better, let me do that.

Comment (Anindyadeep) [reply cookbook/url-topic-summarizer.mdx]: Check now, and leet me know :)

File: cookbook/url-topic-summarizer.mdx (hunk 2 above)
Comment (Anindyadeep) [reply cookbook/url-topic-summarizer.mdx]: Done

**Comment by Anindyadeep** [comment]
Hi I am merging this one, and we can re-iterate on it if required.

**Pull request merged**